pkgs/applications/version-management/git-and-tools/gitstatus/default.nix

```diff
@@ -0,0 +1,28 @@
+{callPackage, stdenv, fetchFromGitHub, ...}:
+
+stdenv.mkDerivation rec {
+  pname = "gitstatus";
+  version = "20190506";
+
+  src = fetchFromGitHub {
+    owner = "romkatv";
+    repo = "gitstatus";
+    rev = "9c791f93c23c04dadfab8b4309a863b62a6ee424";
+    sha256 = "0jbdrgl62x6j920h72n2q6304fb6gdgnmllpv4aa76m13b9qhgq6";
+  };
+
+  buildInputs = [ (callPackage ./romkatv_libgit2.nix {}) ];
+  installPhase = ''
+    install -Dm755 gitstatusd $out/bin/gitstatusd
+    install -Dm444 gitstatus.plugin.zsh $out
+    sed -i "s|local daemon.*|local daemon=$out/bin/gitstatusd|" $out/gitstatus.plugin.zsh
```

Comment:
It would be better to do this in the `patchPhase` if possible.

Reply:
done